source/2019-08-02-the-ember-times-issue-109.md

```diff
@@ -81,12 +82,15 @@ To learn more about exactly how DDAU prevents circular dependencies, we encourag
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<consider adding some bold to your paragraph>
+## [Super Rentals in Octane 🚀](https://github.com/ember-learn/super-rentals/pull/127)
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+If you've been looking for an example of what an app written in **Ember Octane** looks like, look no futher than the rewrite of the Super Rentals Tutorial that's been a part of the Ember.js guides for quite a while now.
+
+The original rewrite PR was opened by [@vaidehijoshi](https://github.com/vaidehijoshi), author of the amazing ✨ [basecs](https://github.com/vaidehijoshi/basecs-series) and [baseds](https://github.com/vaidehijoshi/baseds-series) series. 
+
+[@tomdale](https://github.com/tomdale) also followed up with contributions to get the Octane vesion of Super Rentals over the line.
+
+This new version of the tutorial will become a part of the guides when Ember Octane is released. You can find both PR's [here](https://github.com/ember-learn/super-rentals/pull/127) and [here](https://github.com/ember-learn/super-rentals/pull/129), if you're interested in taking a look.
```

Comment:
Looking at the Octane guides preview site (https://octane-guides-preview.emberjs.com/release/), I guess the tutorial's not live yet, right? To clarify to the readers, maybe:
```suggestion
The new Super Rentals tutorial will become a part of the guides when Ember Octane is released. For a sneak preview, you can check out the PR's [here](https://github.com/ember-learn/super-rentals/pull/127) and [here](https://github.com/ember-learn/super-rentals/pull/129).
```